src/redis-benchmark.c

```diff
@@ -1941,8 +1941,8 @@ int main(int argc, const char **argv) {
         }
 
         if (test_is_selected("lrange") || test_is_selected("lrange_500")) {
-            len = redisFormatCommand(&cmd,"LRANGE mylist%s 0 449",tag);
-            benchmark("LRANGE_500 (first 450 elements)",cmd,len);
+            len = redisFormatCommand(&cmd,"LRANGE mylist%s 0 499",tag);
+            benchmark("LRANGE_500 (first 500 elements)",cmd,len);
```

Comment:
@filipecosta90 @itamarhaber  what do you think of the chance that someone uses this benchmark to spot regressions (running it weekly and storing the results). now we'll change the content of this test from 450 to 500, it'll suddenly run slower.
on the other hand, changing the label can break some script, right?

Reply:
Anything is possible, granted, but I think the risk is low here